presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1175,6 +1175,41 @@ public void testCreateEmptyNonBucketedPartition()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testUnregisterRegisterPartition()
+    {
+        String tableName = "test_register_partition_for_table";
+        assertUpdate("" +
+                "CREATE TABLE " + tableName + " (" +
+                "  dummy_col bigint," +
+                "  part varchar)" +
+                "WITH (" +
+                "  format = 'ORC', " +
+                "  partitioned_by = ARRAY[ 'part' ] " +
```

Comment:
`ORC` is default, so you can put whole `WITH` clause on one line
(also, we would write `['part']` rather than `[ 'part' ]`)